build/assets/tuned/openshift-node-performance

```diff
@@ -25,6 +25,9 @@ governor=performance                          #  latency-performance
 energy_perf_bias=performance                  #  latency-performance 
 min_perf_pct=100                              #  latency-performance 
 
+[service]
+service.stalld=start,enable
```

Comment:
Is it going to work fine with the default arguments? Or should we pass in a config file with env vars?

Reply:
@MarSik  default arguments are set via NTO here :  https://github.com/openshift/cluster-node-tuning-operator/pull/152/files#diff-98484aa29ef1977106bf3b4930e1199a

Reply:
I guess Martin is asking whether there should be a way to specify an environment file to override the default arguments.  You can do this by something like:

```
      [service]
      service.stalld=start,enable,file:/host/etc/tuned/openshift-realtime/stalld.conf
```

However, you'd need to implement a mechanism to place the environment file on the host.  And this is probably what the question is about I'm guessing?

Reply:
If the arguments set in NTO should be the default for performance then the answer should be yes.
We should strive to reduce using machine configs on PAO to keep restarts at minimum. 
But we should decide whether we want to enable this kind of configuration on PAO side

Reply:
In fact I was just asking if the defaults are good enough :) And I think they are.